app/Services/Manager.php

```diff
@@ -189,19 +204,31 @@ public function collectContestInfo($data)
      *
      * @return object $signups
      */
-    public function getUserSignup($id, $campaign = null, $campaignRun = null)
+    public function getUserSignup($id, $campaign = null, $campaignRun = null, $getMultiple = false)
     {
         $signups = $this->northstar->getUserSignups($id, $campaign);
 
+        if ($getMultiple) {
+            $multipleSignups = [];
+        }
```

Comment:
Instead of adding the additional `$getMultiple` param, couldn't you just do a check if the `$id` (which could be renamed to `$ids`) is an array?